### ISSUE TITLE
Capacitor support?

### ISSUE THREAD

**Issue by dylanvdmerwe** [NeutrinosPlatform/cordova-plugin-document-scanner#105]
**Describe the bug**
I wanted to check, and have it officially stated somewhere, if this plugin is supported and works on Capacitor (iOS and Android)?

It would be a good idea to update the readme to state this if it does in fact support Capacitor.

Are there any specific requirements, or steps needed, to make sure that the plugin is setup and working correctly on Capacitor?

**Comment by ChrisTomAlx** [comment]
I have not myself tested it on Capacitor, since I usually use vanilla Cordova. But if you are willing to test it out and raise a pr with a change to the readme, I will gladly take a look and help with merging it. Although do go through the other issues first, if I remember correctly, there have been a few people who tried getting it to work on Capacitor.

Cheers and have a nice day :)
[Chris](https://github.com/ChrisTomAlx)
[Neutrinos](https://goneutrinos.com/)

**Comment by dylanvdmerwe** [comment]
Will test and try. Could be worth adding here: 
https://github.com/danielsogl/awesome-cordova-plugins
https://github.com/riderx/awesome-capacitor

**Comment by dylanvdmerwe** [comment]
@ChrisTomAlx do you recommend trying with latest v5.0.0-beta.1?

**Comment by ChrisTomAlx** [comment]
Yes @dylanvdmerwe. You can try it with the latest beta version. Not an issue.

Cheers and have a nice day :)
[Chris](https://github.com/ChrisTomAlx)
[Neutrinos](https://goneutrinos.com/)
